libtsuba/src/RDGCore.cpp

```diff
@@ -131,6 +132,38 @@ schemify(const std::vector<katana::PropStorageInfo>& prop_info_list) {
   }
   return arrow::schema(fields);
 }
+
+template <typename StorageType>
+katana::Result<katana::NUMAArray<katana::EntityTypeID>>
+LoadAndMaybeConvertTypeIDArray(
+    size_t begin, size_t end, const katana::Uri& types_path,
+    const katana::RDGPartHeader& part_header) {
+  katana::NUMAArray<katana::EntityTypeID> types;
+
+  // NB: we add sizeof(EntityTypeIDArrayHeader) to every range element because
+  // the structure of this file is
+  // [header, value, value, value, ...]
+  // Recent storage formats remove this header
+  size_t header_size = part_header.unstable_storage_format()
```

Comment:
FYI the unstable format is officially becoming version 4 in https://github.com/KatanaGraph/katana/pull/833
which is just having some CI trouble on the enterprise side of things :/ 

The change here would be just like https://github.com/KatanaGraph/katana/pull/833/files#diff-c5c5eb22f4a9863510a9a228eb80a6239b374834360ea096c3366a66e1240330

Reply:
Would you like me to wait to merge after #833?

Reply:
Nah, its not a big deal either way. 
Just ping me if this merges before #833 so I can make sure to fix it up

Reply:
If you don't have any objections, would you approve it?

Reply:
sorry, was mid-review when I got interrupted earlier.